src/blister.c

```diff
@@ -284,17 +284,30 @@ Obj DoCopyBlist(Obj list, Int mut) {
   
 }
 
+Obj CopyBlistImm (
+    Obj                 list,
+    Int                 mut )
+{
+  return list;
+}
+		  
 Obj CopyBlist (
     Obj                 list,
     Int                 mut )
 {
-
-    /* don't change immutable objects                                      */
-    if ( ! IS_MUTABLE_OBJ(list) ) {
-        return list;
-    }
```

Comment:
I added the code in again as to make this sensibly mergeable, because I agree that the check should be there. If @stevelinton comments and gives a reason why it can go, I can remove this again.